package.json

```diff
@@ -1,19 +1,14 @@
 {
-  "dependencies": {
-    "diff-match-patch": "^1.0.5",
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-select": "^2.4.4"
-  },
   "devDependencies": {
     "karma": "^6.3.20",
     "karma-chrome-launcher": "^3.1.1",
     "karma-cljs-test": "^0.1.0",
     "karma-junit-reporter": "^2.0.1",
     "puppeteer": "^14.1.2",
-    "shadow-cljs": "^2.16.6"
-  },
-  "scripts": {
-    "dev": "shadow-cljs watch app"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "shadow-cljs": "^2.16.6",
+    "diff-match-patch": "^1.0.5",
+    "react-select": "^2.4.4"
```

Comment:
updating `react@^17.0.2` seems to cause peer-dependency issue with `react-select@^2.4.4` when running `npm install`:
```sh
...
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.3.0 || ^16.0.0" from react-select@2.4.4
npm ERR! node_modules/react-select
npm ERR!   dev react-select@"^2.4.4" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: react@16.14.0
npm ERR! node_modules/react
npm ERR!   peer react@"^15.3.0 || ^16.0.0" from react-select@2.4.4
npm ERR!   node_modules/react-select
npm ERR!     dev react-select@"^2.4.4" from the root project
...
```

i think we need to look at updating `react-select` to a newer version, but there are quite a few majors between ours and latest.

Reply:
Yes, I added an idea about this. Not sure if it's a problem since everything seems to work ...

Reply:
Ok, seems like the upgrade was easy. Let's test this manually in demo. Issue in #3079.